tools/tertiary-analysis/decoupler/decoupler_pathway_inference.xml

```diff
@@ -15,6 +15,9 @@
             --source $source
             --target $target
             --weight $weight
+            #if $gene_symbols_field:
+                --gene_symbols_field '$gene_symbols_field'
```

Comment:
@anilthanki tests are probably passing because no new test is testing this, but I would expect this to match the python script option:

```suggestion
                --var_gene_symbols_field '$gene_symbols_field'
```

I suspect that the other decoupler methods that use this option might have a test for that?

Reply:
`Aucell` has a test but using different dataset.

Reply:
Then it is free to use it here, you don't need to find it nor download it!

Reply:
If the gene symbols match sufficiently, it should do.

Reply:
Test `adata.var` does not have `gene_symbol` column

Reply:
```
>>> adata.var.columns
Index(['n_cells'], dtype='object')
```

Reply:
if you have some mouse network, you could use this anndata https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/blob/main/tools/tertiary-analysis/decoupler/decoupler_aucell_score.xml#L128